app/controllers/api/custom_button_sets_controller.rb

```diff
@@ -7,5 +7,12 @@ def create_resource(type, id, data)
     def edit_resource(type, id, data)
       super(type, id, data.deep_symbolize_keys)
     end
+
+    def order_resource(_, _, data)
+      CustomButtonSet.update_group_index(data["ids"])
```

Comment:
I think we need an RBAC check here, or does that happen automatically?  cc @kbrock

Reply:
@akhilkr128 Do you know what RBAC was used in the old UI?

Reply:
We need an admin role, correct me if I am wrong. 
see the [yml](https://github.com/ManageIQ/manageiq/blob/619680b323bc4305e23508cb40228225fdc5b85e/db/fixtures/miq_product_features.yml#L2583)